### ISSUE TITLE
Parameter name on the code is different with jsDocs in Client#channelUpdate

### ISSUE THREAD

**Issue by andrerahardjo97** [discordjs/discord.js#5049]
Look at Client#channelUpdate in https://github.com/discordjs/discord.js/blob/stable/src/client/websocket/handlers/CHANNEL_UPDATE.js#L14
The parameter name is `old` and `updated`, but it should be `oldChannel` and `newChannel`.
Because of this, the code completion is not appear as it should be.

Thanks in advance and sorry if my english is bad.

**Comment by mukeshmjn** [comment]
I would like to fix it . Please assign me this

**Comment by mukeshmjn** [comment]
Is this issue fixed its been a month since he commented let me fix this

**Comment by mukeshmjn** [comment]
Where are the jsdocs i cant find it can you guide please?

**Comment by mukeshmjn** [comment]
I bymistake unassigned myself can you reassign please

**Comment by xAtsuUC** [comment]
lmfao

**Comment by mukeshmjn** [comment]
@kyranet yeah it happened by mistake

**Comment by kyranet** [comment]
> The parameter name is `old` and `updated`, but it should be `oldChannel` and `newChannel`.
> Because of this, the code completion is not appear as it should be.

This is incorrect, IntelliSense doesn't work by scanning dozens of thousands (and possibly millions if you have many dependencies) of lines of code, it gets the names from our TypeScript definitions, and I just saw that this issue specifically isn't present anymore in master:

https://github.com/discordjs/discord.js/blob/d744e51c1bdb4c7a26c0faeea1f2f45baaf5fd3c/typings/index.d.ts#L2377

Where the parameters are well-defined with the same name as the JSDocs ones.

This patch points to #5225 as the one that last changed this line, and therefore, that PR has fixed this issue, but did not yet close it.

---

Sorry for the issue assignments, I assumed the issue was still happening as it wasn't closed, at first, I wasn't understanding what the issue was, but I got it! 😅
